packages/core/src/routes/session.ts

```diff
@@ -68,14 +68,13 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     koaGuard({
       body: object({
         username: string(),
-        password: string(),
+        password: string().nonempty(),
```

Comment:
can add nonempty to usename as well.

Reply:
Fixed.
Was thinking that an empty username string would lead to a `username_not_exist` error. It is good to do the check at the very beginning.